### PR TITLE
Roblox Modal content update

### PR DESCRIPTION
**Pull request by adamkecskes**
CodeCombat_-_Coding_games_to_learn_Python_and_JavaScript___CodeCombat

### CODE REVIEW

File: app/styles/modal/mine-modal.sass
Comment (nwinter) [inline app/styles/modal/mine-modal.sass:1]: Let's make this mobile responsive

**Review by nwinter — CHANGES_REQUESTED** [review]

**Comment by nwinter** [comment]
Let's also add another secondary button/link to be able to get to /roblox to link your Roblox account

File: app/views/HomeView.coffee [hunk 1]
```diff
@@ -230,6 +234,8 @@ module.exports = class HomeView extends RootView
         url = new URLSearchParams window.location.search
         _.defer => @openModalView(new AuthModal({initialValues:{email: url.get 'email'}})) unless @destroyed
 
+    _.defer => @openModalView new MineModal() unless storage.load('roblox-clicked') or @destroyed #if utils.getQueryVariable('roblox-clicked') isnt 'true'
```
[the remark this answers is not in the document]
Comment (adamkecskes) [reply app/views/HomeView.coffee:237]: @nwinter don't we have a dedicated HomeCNView?

**Comment by adamkecskes** [comment]
Now it's responsive: 
roblox-modal

**Comment by nwinter** [comment]
Awesome! Maybe for the link to /roblox, we can put it as "Link Roblox Account" as another button with a different color, next to the "Play Now" button?

Ideally it would have a "Learn More" also (which goes to same place), but getting too busy without refactoring more about how the modal works.

**Comment by adamkecskes** [comment]
button added. yeah, three button might be too much.
CodeCombat_-_Coding_games_to_learn_Python_and_JavaScript___CodeCombat

File: app/views/HomeView.coffee (hunk 1 above)
[the remark this answers is not in the document]
Comment (nwinter) [reply app/views/HomeView.coffee]: Oh right, so I guess it doesn't matter

**Review by nwinter — APPROVED** [review]
Nice!

**Pull request merged**